### PR TITLE
add: `mold-bin`

### PR DESCRIPTION
**Pull request by wizard-28**

### CODE REVIEW

File: packages/mold-bin/mold-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+arch=("amd64")
+url="https://github.com/rui314/mold/releases/download/v${version}/mold-${version}-x86_64-linux.tar.gz"
+description="A modern linker"
+breaks=("${name}-git" "$name")
+hash="bf788940db4a9ac19e7745c821bf6ee18ff4d75441a803d84f86c9f3b0aa2a5e"
+repology=("project: mold")
+
+install() {
+	sudo mkdir -p "${pkgdir}/usr/local"
+	sudo mv ./* "${pkgdir}/usr/local"
```
Comment (Henryws) [inline packages/mold-bin/mold-bin.pacscript:20]: Doesn't debian and arch forbid packaging to /usr/local?

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/mold-bin/mold-bin.pacscript (hunk 1 above)
Comment (Henryws) [reply packages/mold-bin/mold-bin.pacscript:20]: Pretty sure it is.

Comment (wizard-28) [reply packages/mold-bin/mold-bin.pacscript:20]: Ok, suggest the change

Comment (Henryws) [reply packages/mold-bin/mold-bin.pacscript:20]: I don't know where it should go. I just know it can't be `/usr/local/`

Comment (Henryws) [reply packages/mold-bin/mold-bin.pacscript:20]: Found it: https://www.debian.org/doc/packaging-manuals/fhs/fhs-3.0.html#usrlocalLocalHierarchy

Comment (Henryws) [inline packages/mold-bin/mold-bin.pacscript:20]:
```suggestion
	sudo mkdir -p "${pkgdir}/usr/"
	sudo mv ./* "${pkgdir}/usr/"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/mold-bin/mold-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#  __      __________   ______
+# /  \    /  \_____  \ /  __  \
+# \   \/\/   //  ____/ >      <
+#  \        //       \/   --   \
+#   \__/\  / \_______ \______  /
+#        \/          \/      \/
+maintainer="wizard-28 <wiz28@pm.me>"
+
+name="mold-bin"
```
Comment (Henryws) [inline packages/mold-bin/mold-bin.pacscript:9]:
```suggestion
name="mold-bin"
gives="mold"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/mold-bin/mold-bin.pacscript [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+gives="mold"
+version="1.11.0"
+arch=("amd64")
+url="https://github.com/rui314/mold/releases/download/v${version}/mold-${version}-x86_64-linux.tar.gz"
```
Comment (Henryws) [inline packages/mold-bin/mold-bin.pacscript:13]: You could add a `case "${CARCH}"` part to make this pacscript available on arm and arm64 as well.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/mold-bin/mold-bin.pacscript (hunk 3 above)
Comment (wizard-28) [reply packages/mold-bin/mold-bin.pacscript:13]: No

Comment (Henryws) [inline packages/mold-bin/mold-bin.pacscript:13]:
```suggestion
case "${CARCH}" in
    amd64)
        url="https://github.com/rui314/mold/releases/download/v${version}/mold-${version}-x86_64-linux.tar.gz"
        hash="bf788940db4a9ac19e7745c821bf6ee18ff4d75441a803d84f86c9f3b0aa2a5e"
        ;;
    arm64)
        url="https://github.com/rui314/mold/releases/download/v${version}/mold-${version}-aarch64-linux.tar.gz"
        hash="4d7f0cc976ed1dfde1ceeaadd16829427a477eba1b49ff82dd490395f32eb78c"
        ;;
esac
```

File: packages/mold-bin/mold-bin.pacscript [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+description="A modern linker"
+breaks=("${name}-git" "$name")
+hash="bf788940db4a9ac19e7745c821bf6ee18ff4d75441a803d84f86c9f3b0aa2a5e"
```
Comment (Henryws) [inline packages/mold-bin/mold-bin.pacscript:16]:
```suggestion
```

File: packages/mold-bin/mold-bin.pacscript [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+#        \/          \/      \/
+maintainer="wizard-28 <wiz28@pm.me>"
+
+name="mold-bin"
+gives="mold"
+version="1.11.0"
+arch=("amd64")
+url="https://github.com/rui314/mold/releases/download/v${version}/mold-${version}-x86_64-linux.tar.gz"
+description="A modern linker"
+breaks=("${name}-git" "$name")
```
Comment (Henryws) [inline packages/mold-bin/mold-bin.pacscript:15]:
```suggestion
breaks=("${name}-git" "${name}")
```

File: packages/mold-bin/mold-bin.pacscript [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+# \   \/\/   //  ____/ >      <
+#  \        //       \/   --   \
+#   \__/\  / \_______ \______  /
+#        \/          \/      \/
+maintainer="wizard-28 <wiz28@pm.me>"
+
+name="mold-bin"
+gives="mold"
+version="1.11.0"
+arch=("amd64")
```
Comment (Henryws) [inline packages/mold-bin/mold-bin.pacscript:12]:
```suggestion
arch=("amd64" "arm64")
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**